pkg/migrations/op_common_test.go

```diff
@@ -400,6 +414,40 @@ func columnHasType(t *testing.T, db *sql.DB, schema, table, column, expectedType
 	return expectedType == actualType
 }
 
+func columnHasComment(t *testing.T, db *sql.DB, schema, table, column, expectedComment string) bool {
+	t.Helper()
+
+	var actualComment string
+	err := db.QueryRow(fmt.Sprintf(`
+    SELECT col_description(
+      %[1]s::regclass,
+      (SELECT attnum FROM pg_attribute WHERE attname=%[2]s and attrelid=%[1]s::regclass)
+    )`,
```

Comment:
should we add this to the read_schema function?

Reply:
Looks like it's already there:

https://github.com/xataio/pgroll/blob/f1757f31a270c593e45b67ea79cd38af5c16cc21/pkg/state/state.go#L105

for table comments 

and 

https://github.com/xataio/pgroll/blob/f1757f31a270c593e45b67ea79cd38af5c16cc21/pkg/state/state.go#L129

for column comments.

Example output for one table:

```json
{
  "employees": {
    "oid": "16629",
    "name": "employees",
    "columns": {
      "id": {
        "name": "id",
        "type": "integer",
        "unique": true,
        "comment": null,
        "default": "nextval('_pgroll_new_employees_id_seq'::regclass)",
        "nullable": false
      },
      "job_title": {
        "name": "job_title",
        "type": "varchar(255)",
        "unique": false,
        "comment": "this is a comment for the role column",
        "default": null,
        "nullable": false
      }
    },
    "comment": "this is a comment for the employees table",
    "indexes": {
      "_pgroll_new_employees_pkey": {
        "name": "_pgroll_new_employees_pkey"
      }
    },
    "primarykey": [
      "id"
    ],
    "foreignkeys": null
  }
}
```